server/src/index.ts

```diff
@@ -35,12 +36,12 @@ const opensrpAuth = new ClientOAuth2({
   scopes: ['read', 'write'],
   state: EXPRESS_OPENSRP_OAUTH_STATE,
 });
-
 const loginURL = EXPRESS_SESSION_LOGIN_URL || '/';
 const sessionName = EXPRESS_SESSION_NAME || 'session';
 
 const app = express();
 
+app.use(cors());
```

Comment:
I don't know what the default configs are for [cors](https://www.npmjs.com/package/cors), but should we be passing in options to limit / control resource requests?